getting-started/ts-example/monitoring.ts

```diff
@@ -1,6 +1,7 @@
 import { MeterProvider } from '@opentelemetry/metrics';
-import { Metric, BoundCounter } from '@opentelemetry/api';
+import { Counter } from '@opentelemetry/api';
 import { PrometheusExporter } from '@opentelemetry/exporter-prometheus';
+import { RequestHandler } from "express";
```

Comment:
interesting is that this line didn't cause the build to fail on linting, as there should be single quotes.

Reply:
I don't think the getting started guide is linted

Reply:
@mickdekkers can you please change this line to use single quotes ?

Reply:
Sure thing, I'll change it to single quotes 👍

FYI, this isn't consistent in the getting started guide: `tracing.ts` uses both single and double quotes and `app.ts` uses double quotes

Reply:
Like I said, the guide isn't linted. Since the guide rarely changes, I think it isn't necessarily a problem. Let's make these small quote changes and merge this. Eventually, the guide will need to be rewritten for the `sdk` package, and maybe we should start linting it then.